.github/workflows/android.yml

```diff
@@ -46,3 +46,34 @@ jobs:
 
         # Build LLM Demo for Android
         bash build/build_android_llm_demo.sh ${{ matrix.tokenizer }} ${ARTIFACTS_DIR_NAME}
+  # Upload artifacts to S3. The artifacts are needed not only by the device farm but also TorchChat
+  upload-artifacts:
+    needs: build-llm-demo
+    runs-on: linux.2xlarge
+    steps:
+      - name: Download the artifacts from GitHub
```

Comment:
Fyi, @huydhn renamed it a bit for better clarifty as I was confused why and where we download the artifacts.